homeassistant/components/glances/coordinator.py

```diff
@@ -43,3 +45,13 @@ async def _async_update_data(self) -> dict[str, Any]:
         except exceptions.GlancesApiError as err:
             raise UpdateFailed from err
         return data or {}
+
+    def get_uptime(self) -> datetime | None:
+        """Get uptime by converting Glances duration to datetime."""
+        uptime = None
+        if self.data:
+            uptime = self.data["uptime"]
+            up_duration = parse_duration(uptime)
+            if up_duration:
+                uptime = utcnow() - up_duration
+        return uptime
```

Comment:
This does not give an uptime. It gives you the time it started.
Should the sensor be named differently?

Reply:
This seems to be the standard in home assistant : uptime sensors return a datetime which is the moment of startup. 
See for example the uptime integration : 
https://github.com/home-assistant/core/blob/bf8d880e5f7d06486879c75ebb920af2e01b5d24/homeassistant/components/uptime/sensor.py#L34
A few other examples:
https://github.com/home-assistant/core/blob/bf8d880e5f7d06486879c75ebb920af2e01b5d24/homeassistant/components/unifi/sensor.py#L111
https://github.com/home-assistant/core/blob/bf8d880e5f7d06486879c75ebb920af2e01b5d24/homeassistant/components/bbox/sensor.py#L143

In the UI, the absolute datetime is displayed as a relative time (like 2 days 10 hours..) by default, so I think it makes sense.